transport-native-epoll/src/main/java/io/netty/channel/epoll/AbstractEpollStreamChannel.java

```diff
@@ -88,6 +75,7 @@
 
     private WritableByteChannel byteChannel;
 
+    @Deprecated
```

Comment:
I would not expect to have a user extend this class so I think its fine to have no replacement, no ?